test/Serializer.js

```diff
@@ -0,0 +1,251 @@
+const chaiAsPromised = require("chai-as-promised");
+const chai = require("chai");
+chai.use(chaiAsPromised);
+const {expect} = chai;
+const dynamoose = require("../lib");
+const {Model} = dynamoose;
+const Serializer = require("../lib/Serializer");
+
+describe("Serializer", () => {
+	let User;
+	beforeEach(() => {
+		User = new Model("User",
+			{id: Number, name: String, email: String, phone: String, passwordHash: String, status: String},
+			{"create": false, "waitForActive": false}
+		);
```

Comment:
I know it's a lot on one line, but I'd prefer to maintain consistency throughout.

```suggestion
		User = new Model("User", {id: Number, name: String, email: String, phone: String, passwordHash: String, status: String}, {"create": false, "waitForActive": false});
```

Reply:
Fixed in e1edde6ca579de801db6063a5dd387cd374cd797